src/main/java/com/spotify/faceapi/Service/TopTrackService.java

```diff
@@ -42,8 +42,18 @@ public Object getTopTracks(String token, int term) throws NoAccountException, Un
             throw new NoTrackException(AppConstants.NoTrackException_Msg);
         }
         return result;
-
-    } catch (Exception e) {
+        } catch (
+                HttpClientErrorException.Unauthorized e) {
+            throw new UnAuthorizedException(AppConstants.UnAuthorizedException_Msg);
+        } catch (HttpClientErrorException.Forbidden e) {
+            throw new ForbiddenException(AppConstants.ForbiddenException_Msg);
+        } catch (HttpClientErrorException.BadRequest e) {
+            throw new BadRequestException(AppConstants.BadRequestException_Msg);
+        } catch (HttpClientErrorException.NotFound e) {
+            throw new NotFoundException(AppConstants.NotFoundException_Msg);
+        } catch (NoTrackException e) {
+            throw new NoArtistsException(AppConstants.NoTrackException_Msg);
```

Comment:
Is this correct?  "no track" -> "no artists"?

If it's not correct, then why aren't your unit tests catching this?